rdiff-backup --version from Ubuntu PPA show DEV

Want to report a problem. I didn't check what wrong with the build.
```
docker run -it --rm ubuntu
apt update && apt install software-properties-common
add-apt-repository ppa:rdiff-backup/rdiff-backup-development
apt update
apt install rdiff-backup
rdiff-backup --version
```
Shows `rdiff-backup DEV`

I can't find where this `VER` string comes from anyway, not in the build Python package at least..

```
rdiff-backup$ grep -rF VER *
usr/lib/python3/dist-packages/rdiff_backup/log.py:            os.getenv('RDIFF_BACKUP_VERBOSITY', '3'))
usr/lib/python3/dist-packages/rdiff_backup/log.py:                role = "SERVER"
```

I tested @ikus060 tip, but it did not work:
```
commit 7c94e01617de599fe35659e83b4efcd8d22e1e2a (HEAD -> dev-otto)
Author: Otto Kekäläinen <otto@debian.org>
Date:   Sat Apr 18 14:38:58 2020 +0300

    WIP: Fix https://github.com/rdiff-backup/rdiff-backup/issues/305

diff --git a/debian/rules b/debian/rules
index 14680c3..761e0a0 100755
--- a/debian/rules
+++ b/debian/rules
@@ -1,4 +1,6 @@
 #!/usr/bin/make -f
 
+export SETUPTOOLS_SCM_PRETEND_VERSION=$(shell python3 setup.py --version)
+
 %:
        dh $@ --buildsystem=pybuild --with python3
```

.deb build:
```
gbp:info: Creating /tmp/build/rdiff-backup_2.0.0.orig.tar.gz
gbp:info: Performing the build
 dpkg-buildpackage -us -uc -ui -i -I
dpkg-buildpackage: info: source package rdiff-backup
dpkg-buildpackage: info: source version 2.0.0-1
dpkg-buildpackage: info: source distribution unstable
dpkg-buildpackage: info: source changed by Otto Kekäläinen <otto@debian.org>
 dpkg-source -i -I --before-build .
dpkg-buildpackage: info: host architecture amd64
 fakeroot debian/rules clean
dh clean --buildsystem=pybuild --with python3
/usr/lib/python3/dist-packages/setuptools_scm/version.py:92: UserWarning: tag 'debian/2.0.0-1' no version found
  warnings.warn("tag %r no version found" % (tag,))
Traceback (most recent call last):
  File "setup.py", line 225, in <module>
    setup_requires=['setuptools_scm'],
  File "/usr/local/lib/python3.7/dist-packages/setuptools/__init__.py", line 145, in setup
    return distutils.core.setup(**attrs)
  File "/usr/lib/python3.7/distutils/core.py", line 108, in setup
    _setup_distribution = dist = klass(attrs)
  File "/usr/local/lib/python3.7/dist-packages/setuptools/dist.py", line 448, in __init__
    k: v for k, v in attrs.items()
  File "/usr/lib/python3.7/distutils/dist.py", line 292, in __init__
    self.finalize_options()
  File "/usr/local/lib/python3.7/dist-packages/setuptools/dist.py", line 740, in finalize_options
    ep.load()(self)
  File "/usr/local/lib/python3.7/dist-packages/setuptools/dist.py", line 747, in _finalize_setup_keywords
    ep.load()(self, ep.name, value)
  File "/usr/lib/python3/dist-packages/setuptools_scm/integration.py", line 17, in version_keyword
    dist.metadata.version = get_version(**value)
  File "/usr/lib/python3/dist-packages/setuptools_scm/__init__.py", line 150, in get_version
    parsed_version = _do_parse(config)
  File "/usr/lib/python3/dist-packages/setuptools_scm/__init__.py", line 98, in _do_parse
    version = _version_from_entrypoints(config) or _version_from_entrypoints(
  File "/usr/lib/python3/dist-packages/setuptools_scm/__init__.py", line 58, in _version_from_entrypoints
    version = _call_entrypoint_fn(root, config, ep.load())
  File "/usr/lib/python3/dist-packages/setuptools_scm/__init__.py", line 39, in _call_entrypoint_fn
    return fn(root, config=config)
  File "/usr/lib/python3/dist-packages/setuptools_scm/git.py", line 135, in parse
    branch=branch,
  File "/usr/lib/python3/dist-packages/setuptools_scm/version.py", line 204, in meta
    assert parsed_version is not None, "cant parse version %s" % tag
AssertionError: cant parse version debian/2.0.0-1
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -22,7 +22,7 @@ python:
 deploy:
   - provider: releases
     file_glob: true
-    file: dist/*.*
+    file: dist/rdiff*.*
     skip_cleanup: true
     draft: true
     token:

--- a/docs/DEVELOP.md
+++ b/docs/DEVELOP.md
@@ -136,6 +136,7 @@ The same pre-requisites as for the installation of rdiff-backup also apply for b
 * librsync 1.0.0 or higher
 * pylibacl (optional, to support ACLs)
 * pyxattr (optional, to support extended attributes) - even if the xattr library (without py) isn't part of our CI/CD pipeline, feel free to use it for your development
+* python3-setuptools (for a proper version instead of DEV)
 
 Additionally are following pre-requisites needed:
 

--- a/setup.py
+++ b/setup.py
@@ -222,5 +222,6 @@ setup(
         'build_py': build_py,
         'clean': clean,
     },
+    install_requires=['setuptools'],
     setup_requires=['setuptools_scm'],
 )

--- a/tools/build_wheels.sh
+++ b/tools/build_wheels.sh
@@ -12,7 +12,7 @@ for PYBIN in $pybindirs; do
 done
 
 # Bundle external shared libraries into the wheels
-for whl in dist/*.whl; do
+for whl in dist/rdiff_backup*.whl; do
     auditwheel repair "$whl" --plat $PLAT -w /io/dist/
 done
 

--- a/tox.ini
+++ b/tox.ini
@@ -15,6 +15,7 @@ envlist = py35, py36, py37, py38, flake8
 # user being correctly identified (which might not happen in a container)
 passenv = RDIFF_TEST_* RDIFF_BACKUP_*
 deps =
+    setuptools
     pyxattr
     pylibacl
 # whitelist_externals =

--- a/tox_dist.ini
+++ b/tox_dist.ini
@@ -15,6 +15,7 @@ envlist = py35, py36, py37, py38
 passenv = RDIFF_TEST_* RDIFF_BACKUP_*
 skip_install = true
 deps =
+    setuptools-scm
     pyxattr
     pylibacl
     wheel

--- a/tox_root.ini
+++ b/tox_root.ini
@@ -20,6 +20,7 @@ isolated_build_env = .package.root
 # or explicitly the RDIFF_* ones:
 passenv = SUDO_USER SUDO_UID SUDO_GID RDIFF_TEST_* RDIFF_BACKUP_*
 deps =
+    setuptools
     pyxattr
     pylibacl
 #whitelist_externals = env

--- a/tox_slow.ini
+++ b/tox_slow.ini
@@ -12,6 +12,7 @@ envlist = py35, py36, py37, py38
 [testenv]
 passenv = RDIFF_TEST_* RDIFF_BACKUP_*
 deps =
+    setuptools
     pyxattr
     pylibacl
 # whitelist_externals =

